tests/integration/fuzzing/parameterized_test.cairo

```diff
@@ -0,0 +1,25 @@
+%lang starknet
+
+from starkware.cairo.common.math import assert_le
+
+@external
+func setup_examples() {
+    %{
+        max_examples(5)
```

Comment:
[Tests] What happens when `max_examples` is lower than usage of `example`?

Reply:
`max_examples` does not apply to `example`-caused runs. it only affects `given`.

This is all controlled by a concept of _phases_ in Hypothesis. `example` and `given` control separate phases. Another one is shrinking.

https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.Phase

Reply:
`max_examples` only refers to `given`, `example` works separately.

Reply:
It's worth to point out that later in the docs.

Reply:
[update docs](https://github.com/software-mansion/protostar/pull/947/commits/dd62519a5048383692bf43306551af778b51bb28)